packages/webpack/test/fixtures/main/webpack.config.js

```diff
@@ -73,6 +74,18 @@ function makeConfig(lmOptions = {}) {
           ],
           sideEffects: true,
         },
+        {
+          // this explicitly allows svg assets to be emited from dependencies
+          test: /\.svg$/,
+          use: [
+            {
+              loader: 'file-loader',
+              options: {
+                name: '[name].[ext]',
```

Comment:
I know this is just a fixture, but what happens when the filename is `bar.baz.svg`?

Reply:
this part is giving a permission for the file to be emitted. nothing bad can come out of it not covering all cases.